docs/html/user_guide.rst

```diff
@@ -469,20 +478,20 @@ pip's command line options can be set with environment variables using the
 format ``PIP_<UPPER_LONG_NAME>`` . Dashes (``-``) have to be replaced with
 underscores (``_``).
 
-For example, to set the default timeout::
-
-    export PIP_DEFAULT_TIMEOUT=60
-
-This is the same as passing the option to pip directly::
+For example, setting ``PIP_DEFAULT_TIMEOUT`` to 60 has the same effect as
+passing the option to pip directly::
```

Comment:
I feel the previous rendering (using code blocks) is more readable than prose form.

Reply:
I have the same feeling too, but I tried to avoid variable setting syntax because I couldn't find anything that is portable, e.g. [PowerShell uses the hideous `$Env:<variable-name> = "<new-value>"`](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_environment_variables).

Reply:
You can use `.. code-block:: none` to have a code block without syntax highlighting.

Reply:
> Maybe it would help to demonstrate this in Python?

Probably, but I'm not sure how to phrase the surrounding text to interject `os.environ['PIP_FOO'] = bar` that would fit into the comparison with `pip --foo=bar ...`.  I think waiting for https://github.com/pypa/pip/pull/8589#issuecomment-662054242 might be a better idea, or doing a follow-up after it gets merged, depending on which PR will get merged first.

Reply:
Honestly, I've never heard a Windows user complain about being confused by

```
export PIP_DEFAULT_TIMEOUT=60
```

I don't think the issue is important enough to warrant changing things, let's just stick with what we've been using up to now.

Reply:
As somebody who doesn't use Windows, I am delighted to revert back to using `export` :smile: